docs/spring-boot-runtime/master.adoc

```diff
@@ -14,6 +14,20 @@ include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
 {SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, for example, the Embedded Tomcat Container and the Java container image, are provided as part of a Red Hat subscription when used with {SpringBoot}.
 
+The features below are available as a collection of missions and quickstarts link:{link-launcher-oso}[ on {OpenShiftOnline}] or your {OpenShiftLocal}.
+
+.Features and Frameworks Summary
+[format="csv", options="header"]
+[cols="3,10,3,6"]
+|===
+Feature,Problem Addressed, Native, Framework
+Circuit Breaker, Switch between services and continue to operate incoming requests without interruption,No, Spring Cloud Kubernetes - Hystrix
+Health Check, Check readiness and liveness of the service based on information exposed by Spring Boot Actuator. Service restarts automatically if probing fails., Yes, Spring Boot Actuator
+Service Discovery, Discover Service/Endpoint deployed on OpenShift and exposed behind a service or route using the service name matching a DNS entry, Yes - using Kubernetes API, -
+Server Side Load Balancing, Multiple instances of a service deployed will be managed by the Kubernetes Load balancer in a round-robin fashion, Yes - using internal Kubernetes Load Balancer, -
+Client Side Load Balancing, Discover services at the client side using the Server Side Load Balancing feature, No, Spring Cloud Kubernetes - Ribbon
+Externalize Parameters, Externalize application parameters to make the application independent of the environment where it runs, Yes - Kubernetes ConfigMap or Secret, Spring Cloud Kubernetes - ConfigMap
+|===
 
```

Comment:
You need to reformat this table to be consistent with with how we use tables elsewhere in the docs.

For example:
https://github.com/fabric8-launch/appdev-documentation/blob/master/docs/topics/secured-mission-intro.adoc